app/(tabs)/user/index.tsx

```diff
@@ -51,6 +54,22 @@ export default function UserScreen() {
 
         fetchUser();
         console.log(JSON.stringify(user, null, 2));
+
+        const fetchPolls = async () => {
```

Comment:
This whole function has to be called from within the fetch user function. Because use effect is not an async function, fetchUser is not guaranteed to be done before you make the fetch request, which is causing the default user of 1 to be consistently loaded on my end, instead of the account I'm signed in as, I copy and pasted lines 58-72 right after line 52 and it fixed it